crates/subnetworks/history/src/storage_migration.rs

```diff
@@ -0,0 +1,173 @@
+use ethportal_api::{
+    types::{
+        content_value::{
+            history::HistoryContentValue as OldHistoryContentValue,
+            history_new::HistoryContentValue as NewHistoryContentValue,
+        },
+        execution::{header_with_proof as old_hwp, header_with_proof_new as new_hwp},
```

Comment:
```suggestion
        execution::{header_with_proof::BlockHeaderProof as OldBlockHeaderProof, header_with_proof_new::{BlockProofHistoricalHashesAccumulator, BlockHeaderProof, BlockProofHistoricalRoots, BlockProofHistoricalSummaries, HeaderWithProof}},
```

old_hwp is only used for `header_with_proof::BlockHeaderProof as OldBlockHeaderProof`, so I think we can pretty easily avoid using `hwp`